src/ContentValidationListener.php

```diff
@@ -379,7 +379,7 @@ protected function getInputFilterService($controllerService, $method, $isCollect
             return $this->config[$controllerService][$method];
         }
 
-        if (in_array($method, $this->methodsWithoutBodies)) {
+        if (in_array($method, $this->methodsWithoutBodies) || $method === 'DELETE') {
```

Comment:
`$method === 'DELETE'` condition execution costs less than the previous one, so we can swap order here

Reply:
Good point, thank you. Will fix it ASAP